cmd/client-s3.go

```diff
@@ -168,7 +183,9 @@ func newFactory() func(config *Config) (Client, *probe.Error) {
 			// Cache the new minio client with hash of config as key.
 			clientCache[confSum] = api
 		}
-
+		if numParallelThreads := getNumMultiPartThreads(); numParallelThreads > 0 {
+			api.NumParallelWorkers = numParallelThreads
```

Comment:
@poornas, `api.NumParallelWorkers` is undefined, do you need to update minio-go ?